crates/holochain/src/conductor/kitsune_host_impl.rs

```diff
@@ -32,6 +34,8 @@ pub struct KitsuneHostImpl {
     ribosome_store: RwShare<RibosomeStore>,
     tuning_params: KitsuneP2pTuningParams,
     strat: ArqStrat,
+    lair_tag: Option<Arc<str>>,
+    lair_client: Option<lair_keystore_api::LairClient>,
```

Comment:
hmm kitsune needs to be integrated with lair now?

also, should these Options go together? Do they both need to be Some or None together or can they be independent?

Reply:
They *could* go together, but there are also some small options if they are separate:
- if you specify the lair_client and not the tag, you'll get a new node id each time. maybe useful?
- if you specify the tag and not the client, you'll always know which tag is the node id, even in new lair clients. maybe useful?

not sure : )

Reply:
> hmm kitsune needs to be integrated with lair now?

Yeah, this is the most expeditious path forward. If we want to remove that specific dependency in the future, we can write an abstraction, but I don't think it's a high priority.